cmd/mattermost/commands/webhook.go

```diff
@@ -23,6 +24,14 @@ var WebhookListCmd = &cobra.Command{
 	RunE:    listWebhookCmdF,
 }
 
+var WebhookCreateIncomingCmd = &cobra.Command{
+	Use:     "create-incoming",
+	Short:   "Create incoming webhook",
+	Long:    "create incoming webhook",
```

Comment:
Could you please make the `Long` form be more descriptive?

Reply:
👍